src/main/scala/io/movies/Main.scala

```diff
@@ -0,0 +1,56 @@
+package io.movies
+
+import cats.effect._
+import org.typelevel.log4cats.SelfAwareStructuredLogger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+
+import doobie._
+import doobie.hikari._
+import doobie.implicits._
+import io.movies.services.MovieService
+import io.movies.algebras.Repository
+import io.movies.interpreters.RepositoryImpl
+import io.movies.api.MovieServiceHttp4s
+
+object Main extends IOApp {
+  val transactor: Resource[IO, HikariTransactor[IO]] =
+    for {
+      ce <- ExecutionContexts.fixedThreadPool[IO](32) // our connect EC
+      be <- Blocker[IO] // our blocking EC
+      xa <- HikariTransactor.newHikariTransactor[IO](
+        "org.postgresql.Driver", // driver classname
+        //"jdbc:h2:mem:test;DB_CLOSE_DELAY=-1", // connect URL
+        "jdbc:postgresql://127.0.0.1:5432/MovieService",
+        "postgres", // username
+        "somePassword", // password
```

Comment:
The database connection info should be loaded from a configuration file. Please, check out the [pureconfig](https://pureconfig.github.io) library to load the configuration into a case class.

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/c8db88a36b56b6bf0d1a5894ca330ad9aa499565